src/prompt/default.rs

```diff
@@ -77,7 +77,20 @@ impl DefaultPrompt {
 
 fn get_working_dir() -> Result<String, std::io::Error> {
     let path = env::current_dir()?;
-    Ok(path.display().to_string())
+    let path_str = path.display().to_string();
+    let homedir: String = match env::var("USERPROFILE") {
+        Ok(win_home) => win_home,
+        Err(_) => match env::var("HOME") {
+            Ok(maclin_home) => maclin_home,
+            Err(_) => path_str.clone(),
```

Comment:
This would in the failure case always result in `~`?

Reply:
I tested it, by changing `USERPROFILE` to `USERPROFILE2` which doesn't exist and on WSL I changed `HOME` to `HOME2` which doesn't exist. Help me understand what you mean please.